cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -142,7 +142,7 @@ func (mh *MessageHandle) HandleServer(container *mux.MessageContainer, writer mu
 			mh.MessageAcks.Delete(container.Message.Header.ParentID)
 		}
 		return
-	} else if container.Message.GetOperation() == "upload" && container.Message.GetGroup() == modules.UserGroup {
```

Comment:
Seems this's `upload`, not `update`.

Reply:
I add a new constant

Reply:
Looks good, we also have sme hard code for `upload` in eventbus, could you please also fix them?

Reply:
ok.I will modify it in the next pr

Reply:
Thanks, looks good, could you please squash the commits?

Reply:
Ok.Please check it